WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -282,37 +315,35 @@ public void onRefreshStarted() {
                 });
     }
 
-    private @Nullable PostsListAdapter getPostListAdapter() {
+    private @NonNull PostsListAdapter getPostListAdapter() {
```

Comment:
I noticed this method never returned null and assumed the `@Nullable` annotation was a mistake. Nullable would make sense as an annotation to the field, but not this getter method. The changes from using `mPostsListAdapter` directly came later.